packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/index.tsx

```diff
@@ -341,9 +341,9 @@ class StepFetchDevfile extends AbstractLoaderStep<Props, State> {
           <ExpandableWarning
             textBefore="Looking for a Devfile in the git repository failed."
             errorMessage={helpers.errors.getMessage(error)}
-            textAfter="The git provider may not be supported by Eclipse Che (or the git server hasn't been configured
-            appropriately). Note that the git cloning of the repository will succeed if it's public or it's private
-            and the developer git credentials secret has been configured."
+            textAfter="The git provider is either not supported or has not been configured correctly.
+            Note that the git cloning of the public repository will succeed.
+            If the repository is private a Personal Access Token secret should be configured."
```

Comment:
If you want, we can also mention the OAuth option here.

Reply:
I don't think so because a regular user doesn't have an access to configure OAuth.